meson.build

```diff
@@ -0,0 +1,55 @@
+project('nnstreamer', 'c', 'cpp',
+    version: '0.0.3',
+    license: ['LGPL'],
+    meson_version: '>=0.29.0',
```

Comment:
> '>=0.29.0',

Currently, Most of the gstreamer packages and gstreamer-based tools require >=0.47. 
As we all know, NNstreamer is based on Gstreamer. Please check which number do we have to prepare the minimum version of Meson to build NNStreamer in advance.

Reply:
It is right. change version to >= 0.47.0